components/Post.php

```diff
@@ -101,6 +101,17 @@ protected function loadPost()
             return $this->controller->run('404');
         }
 
+        if($post->categories->count() && $this->param('category')) {
+
+            if(false === $post->categories->pluck('slug')->search($this->param('category'))) {
```

Comment:
```suggestion
            if ($post->categories->pluck('slug')->search($this->param('category')) === false) {
```
No yoda statements please 😄